File: launchdarkly/ruby/src/DeleteRepositoryExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    LaunchDarklyClient::CodeReferencesApi.new.delete_repository(
        "repo_string", # repo
    )
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling CodeReferencesApi#delete_repository: #{e}"
end

File: launchdarkly/python/src/DeleteRepositoryExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        api.CodeReferencesApi(api_client).delete_repository(
            repo=None,
        )
    except ApiException as e:
        print("Exception when calling CodeReferencesApi#delete_repository: %s\n" % e)
